modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/metrics/ossm-metrics-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-only_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh
+
+// This is WITHOUT Kiali.
+// TP1 content. TP banner is persistent and handled by Tim O'Keefe in a different PR.
+// Possible file name may change
+// Possible assembly file may change
+// Assemblies, topic map info needs to be worked out still for 3.0.
+
+The following steps show how to integrate {SMProductName} with user-workload monitoring.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* User-workload monitoring is enabled. See link:https://docs.openshift.com/container-platform/4.16/observability/monitoring/enabling-monitoring-for-user-defined-projects.html[Enabling monitoring for user-defined projects].
+
+.Procedure
+
+. Apply a `ServiceMonitor` object to monitor the Istio control plane:
++
+.Example `ServiceMonitor` object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system
+spec:
+  targetLabels:
+  - app
+  selector:
+    matchLabels:
+      istio: pilot
+  endpoints:
+  - port: http-monitoring
+    interval: 30s
+----
+
+. Apply a `PodMonitor` object to collect metrics from Istio proxies:
++
+.Example `PodMonitor` object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: istio-proxies-monitor
+  namespace: istio-system # <1>
+spec:
+  selector:
+    matchExpressions:
+    - key: istio-prometheus-ignore
+      operator: DoesNotExist
+  podMetricsEndpoints:
+  - path: /stats/prometheus
+    interval: 30s
+    relabelings:
+    - action: keep
+      sourceLabels: [__meta_kubernetes_pod_container_name]
+      regex: "istio-proxy"
+    - action: keep
+      sourceLabels: [__meta_kubernetes_pod_annotationpresent_prometheus_io_scrape]
+    - action: replace
+      regex: (\d+);(([A-Fa-f0-9]{1,4}::?){1,7}[A-Fa-f0-9]{1,4})
+      replacement: '[$2]:$1'
+      sourceLabels: [__meta_kubernetes_pod_annotation_prometheus_io_port,
```

Comment:
The values in brackets do not highlight correctly on the preview link: 

![Screenshot from 2024-08-30 12-26-23](https://github.com/user-attachments/assets/5d140985-0a26-4d93-9837-c37e36c21b7d)

I've not seen brackets in use like this, maybe parentheses should be used?

I'd recommend punching the code block into a YAML linter (https://www.yamllint.com/) and seeing the output: 

![Screenshot from 2024-08-30 12-27-46](https://github.com/user-attachments/assets/db088125-62c0-496e-8313-c210d7681d71)

Reply:
The `" "` present in the source https://github.com/istio-ecosystem/sail-operator/tree/main/docs#scraping-metrics-using-the-openshift-monitoring-stack somehow got dropped/removed.  

Checked https://github.com/istio-ecosystem/sail-operator/tree/main/docs#scraping-metrics-using-the-openshift-monitoring-stack against the file in this PR, added back missing `" "` and ran it against YAML linter, and it checked out. 

Have asked Dev to double-check/verify.